roles/cs.nginx-https-termination/templates/vhost.conf.j2

```diff
@@ -36,12 +36,10 @@ server {
     listen 80;
 {% endif %}
 
-    listen 443 ssl http2;
+    {{ ssl_config(vhost) }}
```

Comment:
@pinkeen Haven't you been adding something similar to master already?

Reply:
This is strange, this commit is already on `develop` branch. I don't know how it got into this MR